workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -450,18 +450,36 @@ export default class DynamoDBService {
   }
 
   public transactEdit(params?: {
-    addPutRequest?: Record<string, unknown>[];
+    addPutRequests?: {
+      item: Record<string, unknown>;
```

Comment:
I think we can probably scope these as `Record<string, JsonValue>` instead of `unknown`

Reply:
JSONValue does not encapsulate Sets which is supported by DynamoDB